docs/utils/SolidityLexer.py

```diff
@@ -54,11 +54,10 @@ class SolidityLexer(RegexLexer):
              r'(<<|>>>?|==?|!=?|[-<>+*%&\|\^/])=?', Operator, 'slashstartsregex'),
             (r'[{(\[;,]', Punctuation, 'slashstartsregex'),
             (r'[})\].]', Punctuation),
-            (r'(for|in|while|do|break|return|continue|switch|case|default|if|else|'
-             r'throw|try|catch|finally|new|delete|typeof|instanceof|void|'
-             r'this|import|mapping|returns|private|public|external|internal|'
-             r'constant|memory|storage|payable)\b', Keyword, 'slashstartsregex'),
-            (r'(var|let|with|function|event|modifier|struct|enum|contract|library)\b', Keyword.Declaration, 'slashstartsregex'),
+            (r'(anonymous|break|constant|continue|else|external|if|indexed|internal|'
```

Comment:
This is missing a couple of keywords, `as` and `assembly` to name a few. See the keywords section in `Token.h`.